### ISSUE TITLE
Make sure everything is up to date with the new plugin and make a release

### ISSUE THREAD

**Issue by ajhager** [roboscala/sbt-robovm#35]
I am creating this issue to foster a discussion about @Darkyenus updates to the plugin and the best way to go about releases. As a member of the RoboVM team, I want to make sure sbt-robovm gets the attention it deserves.

I have been investigating using https://jitpack.io/ as an alternative way to distribute sbt-robovm so that releases can be faster and easier.

Feel free to discuss anything about how we can manage the plugin better moving forward.

**Comment by Darkyenus** [comment]
Jitpack certainly looks interesting. One thing though: with publishing to default sbt plugin repo, users didn't have to add a repo for sbt to search in. If I understand correctly, with Jitpack they will have to? It probably isn't a big problem, but still something worth considering I think. What about releases (when RoboVM updates) to sbt repo and Jitpack for snapshots?

Anyway, before the release, we should probably rewrite or at least change the README a bit. It started getting a bit messy when I was incrementally adding new info, so it is not as clear as it could be. And more importantly, there are still links referencing my forks, which is no longer necessary.

Also, somebody who has RoboVM license should check if the debugger works. I tried to add it during the time it was released, everything should be there, but I have never seen it working. And if it works, there should be instructions in README.

In the future, it would be worth writing a more comprehensive documentation, probably on the wiki. I can contribute to that.

And as a reminder, this is a good time to do any refactoring/changes which are breaking. So if somebody sees any architectural problem with the code, speak up now.

Btw. Congratulations on the marriage n' stuff and welcome back, thanks for coming
